interface/patient_file/summary/stats_full.php

```diff
@@ -100,6 +100,9 @@ function dopclick(id, category , type) {
 
     // function to open izi-modal
     function  initIziLink(link , width , height, category, type) {
+        if(category == "medical_problem"){
+            category = "medical problem";
```

Comment:
Can you capitalize this by the way what of a generic function to strip the underscores and replace with space then capitalize. That should work for all the categories.

Reply:
ok